docs/s3cmd-with-minio.md

```diff
@@ -1,105 +1,176 @@
+`S3cmd` is an open source CLI client for managing data in AWS S3, Google Cloud Storage, or any cloud storage service provider that uses the **s3** protocol. `S3cmd` is distributed under the [GPLv2 License](https://www.gnu.org/licenses/old-licenses/gpl-2.0.en.html).
 
-In this recipe we will learn how to configure and use S3cmd to manage data with Minio Server.
+This guide describes how to configure `S3cmd` and use it to manage data on Minio Server. These are the steps you will follow:
 
-## 1. Prerequisites
+1. [Install Minio Server](#installminioserver) 
+2. [Install `S3cmd`](#installs3cmd) 
+3. [Configure `S3cmd`](#configures3cmd) 
+4. [Examples of Typical `S3cmd` Commands](#runs3cmdcommands)
 
-Install Minio Server from [here](http://docs.minio.io/docs/minio-quickstart-guide).
 
-## 2. Installation
+## <a name="installminioserver"></a>1. Install Minio Server
 
-Install `S3cmd` from <http://s3tools.org/s3cmd>.
+Install Minio Server using the instructions in the [Minio Quickstart Guide](http://docs.minio.io/docs/minio-quickstart-guide).
 
-## 3. Configuration
+## <a name="installs3cmd"></a>2. Install `S3cmd`
 
-We will run `S3cmd` on <https://play.minio.io:9000>.
+Install `S3cmd` using these instructions: <http://s3tools.org/s3cmd>.
 
-Access credentials shown in this example belong to <https://play.minio.io:9000>. These credentials are open to public. Feel free to use this service for testing and development. Replace with your own Minio keys in deployment.
+## <a name="configures3cmd"></a>3. Configure `S3cmd`
 
-Edit the following fields in your s3cmd configuration file `~/.s3cfg`
+### 3.1 Generate a Configuration file
+`S3cmd` uses a configuration file called **.s3cfg** to access cloud storage. Use the following command to generate the initial version of **.s3cfg**:
 
 ```sh
-# Setup endpoint
+ `./S3cmd --configure`
+```
+
+**Note:** Accept the defaults when prompted.
+
+### 3.2 Edit the Configuration File
+Modify the configuration file to enable `S3cmd` to manage buckets on https://play.minio.io:9000:
+
+#### 3.2.1. Navigate to **/users/<your user name>** and open **.s3cfg** in a text editor.
+#### 3.2.2. Edit the following fields in **.s3cfg** to configure the endpoint:
+
+```sh
+bucket_location = us-east-1
 host_base = play.minio.io:9000
 host_bucket = play.minio.io:9000
-bucket_location = us-east-1
 use_https = True
+```
 
-# Setup access keys
+#### 3.3.3. Edit the following fields in **.s3cfg** to set the access keys:
+
+```sh
 access_key =  Q3AM3UQ867SPQQA43P2F
 secret_key = zuf+tfteSlswRu7BJ86wekitnifILbZam1KYY3TG
+```
+
+#### 3.3.4. Edit the following fields in **.s3cfg** to enable S3 v4 signature APIs:
 
-# Enable S3 v4 signature APIs
+```sh
 signature_v2 = False
 ```
 
-## 4. Commands
+#### 3.3.5. Save **.s3cfg**.
 
-### To make a bucket
+**Note:** The variables set in this example are for public testing and development on <https://play.minio.io:9000>. Modify these variables as appropriate when developing for your own Minio Server.
```

Comment:
James, can "Modify these variables as appropriate when developing for your own Minio Server." be expressed more clearly. I find that this is the part which is often not understood by new devs.  Replace with your own keys is also not succinct and clear.

Reply:
@deekoder  Specified using "modify ... as appropriate for a development server" and used 3rd person.